target/linux/ath79/nand/base-files/etc/board.d/02_network

```diff
@@ -8,6 +8,11 @@ ath79_setup_interfaces()
 	local board="$1"
 
 	case "$board" in
+	glinet,gl-ar750s-nor |\
+	glinet,gl-ar750s-nor-nand)
+		ucidef_add_switch "switch0" \
+			"0@eth0" "2:lan:2" "3:lan:1" "1:wan"
+		;;
```

Comment:
Same thing with base-files as for 300m, also for cases below.

Reply:
(previously discussed)